src/ontology/mondo-edit.obo

```diff
@@ -176561,7 +176559,7 @@ xref: MESH:C535776 {source="MONDO:equivalentTo"}
 xref: MESH:C537494 {source="Orphanet:166100", source="MONDO:equivalentTo", source="MONDO:preferredExternal", source="Orphanet:166100/e"}
 xref: NORD:1533 {source="MONDO:NORD"}
 xref: OMIM:184840 {source="Orphanet:166100", source="MONDO:equivalentTo", source="Orphanet:166100/e"}
-xref: OMIM:277610 {source="MONDO:equivalentObsolete", source="Orphanet:3450/e", source="MONDO:preferredExternal", source="Orphanet:3450"}
+xref: OMIM:277610 {source="MONDO:equivalentObsolete", source="Orphanet:3450/e", source="Orphanet:3450"}
```

Comment:
I removed `MONDO:preferredExternal` since this is marked as `MONDO:equivalentObsolete`.